wrappers/windres-wrapper.sh

```diff
@@ -1,3 +1,183 @@
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# For more information, please refer to <http://unlicense.org/>
+
+PROG="$0"
+
+CC=${CC:-i686-w64-mingw32-clang}
+
+print_version () {
+    cat <<EOF >&2
+version: ?xd
+EOF
+    exit 1
+}
+
+print_help () {
+ cat <<EOF >&2
+usage: llvm-windres <OPTION> [INPUT-FILE] [OUTPUT-FILE]
+
+LLVM Tool to manipulate Windows resources with a GNU windres interface.
+
+Options:
+  -i, --input <arg>          Name of the input file.
+  -o, --output <arg>         Name of the output file.
+  -J, --input-format <arg>   Input format to read.
+  -O, --output-format <arg>  Output format to generate.
+  --preprocessor <arg>       Custom preprocessor command.
+  --preprocessor-arg <arg>   Preprocessor command arguments.
+  -F, --target <arg>         Target for COFF objects to be compiled for.
+  -I, --include-dir <arg>    Include directory to pass to preprocessor and resource compiler.
+  -D, --define <arg[=val]>   Define to pass to preprocessor.
+  -U, --undefine <arg[=val]> Undefine to pass to preprocessor.
+  -c, --codepage <arg>       Default codepage to use when reading an rc file (0x0-0xffff).
+  -v, --verbose              Enable verbose output.
+  -V, --version              Display version.
+  -h, --help                 Display this message and exit.
+Input Formats:
+  rc                         Text Windows Resource
+  res                        Binary Windows Resource
+Output Formats:
+  res                        Binary Windows Resource
+  coff                       COFF object
+Targets:
+  armv7-w64-mingw32
+  aarch64-w64-mingw32
+  i686-w64-mingw32
+  x86_86-w64-mingw
+EOF
+    exit 1
+}
+
+error() {
+    echo "${PROG}: error: $1" >&2
+    exit 1
+}
+
+quote() {
+    echo "$1" | sed -e "s|'|'\\\\''|g; 1s/^/'/; \$s/\$/'/"
+}
+
+INCLUDE=
+VERBOSE=${VERBOSE:-false}
+INPUT=-
+OUTPUT=/dev/stdout
+INPUT_FORMAT=rc
+OUTPUT_FORMAT=res
+TARGET=X64
+CODEPAGE=
+
+while [ $# != 0 ]; do
+    case "$1" in
+        "-i="*|"--input="*)
+            INPUT="${1#*=}";;
+        "-i"|"--input")
+            INPUT="${2}"; shift;;
+        "-o="*|"--output="*)
+            OUTPUT="${1#*=}";;
+        "-o"|"--output")
+            OUTPUT="${2}"; shift;;
+        "-J="*|"--input-format="*)
+            INPUT_FORMAT="${1#*=}";;
+        "-J"|"--input-format")
+            INPUT_FORMAT="${2}"; shift;;
+        "-O="*|"--output-format="*)
+            OUTPUT_FORMAT="${1#*=}";;
+        "-O"|"--output-format")
+            OUTPUT_FORMAT="${2}"; shift;;
+        "-F="*|"--target="*)
+            TARGET="${1#*=}";;
+        "-O"|"--output-format")
```

Comment:
This should be `-F` and ` --target` right?

Reply:
Yes. Will fix